docs/blog/chibitronics/v0.1.40.md

```diff
@@ -0,0 +1,107 @@
+# Chibitronics Editor v0.1.40
+
+**Posted on October 8, 2017 by [microsoftsam](https://github.com/microsoftsam)**
+
+We are very excited about all the new features we're unveiling this release. 
+This update brings **Scratch** style rendering of the blocks, a suite of new field editors to enhance the editing experience, a new home screen, a number of simulator fixes and much much more.
+
+
+## Scratch blocks
+
+In this release, we have re-vamped the look of our blocks and aligned with **Scratch**. 
+
+Besides the change in the aesthetic look of the blocks, the main difference here is that the Blockly input (reporter) puzzle piece no longer exists. 
+
+![Scratch blocks before/after comparison](/static/blog/chibitronics/v0.1.40/scratchblockschange.png)
+
+All inputs either have a ROUND or HEXAGONAL shape. 
+If a block has a return type, it will have a shape: 
+- Boolean: HEXAGONAL
+- Number, String, Other: ROUND
+
+![Scratch blocks different shaped blocks](/static/blog/chibitronics/v0.1.40/scratchblocksshapes.png)
+
+
+We've based our open source Blockly fork (pxt-blockly) against [scratch-blocks](https://github.com/LLK/scratch-blocks). 
+
+## New Blocks
+
+We have introduced two new APIs to make the editing experience even simpler. 
+
+**turn PIN ON** block: 
+The turn PIN ON/OFF block can be used to switch a pin ON or OFF. Under the hood, the API sets the pin output mode to INPUT / OUTPUT and calls digitalWrite with either 0 or 1.
+
+The Toggle field editor on the block makes it easy to switch between on and off inline.
+
+![turn ON/OFF block screenshot](/static/blog/chibitronics/v0.1.40/turnonoff.png)
+
+
+**show effect** block: 
+The show effect block has a number of pre-programmed effects at it's disposal. You can select the effect you want to use and the pin you want to apply it on, and the built-in effect will run on that PIN. 
+Today we have: 
+- Twinkle
+- Heartbeat
+- Blink
+- SOS
+
+Be sure to check them out!
+
+![show effect block screenshot](/static/blog/chibitronics/v0.1.40/showeffectblock.png)
+
+
+## Deprecated blocks
+
+We have deprecated the **set ON** and **set OFF** blocks in favour of the new **turn ON/OFF** block. They will still work, but you won't be able to create new ones.
+
+## Field editors
+
+Over the past few months, we have invested heavily in our suite of field editors. Today we are happy to include 5 new field editors to the list that can be used by targets.
+
+### String fields: 
+
+All inputs of type string use the string field editor, to allow us to distinguish between a String and a Number. A string field editor shows quotes around the input.
+
+![Field string screenshot](/static/blog/chibitronics/v0.1.40/field_string.png)
+
+### Number dropdown:
+The number dropdown field allows you to edit a number value but also gives the user a list of options to hint to them the sort of values they could expect to use.
```

Comment:
You mean this?:

The number dropdown field allows you to edit a number value but also gives you a list of options which are hints for the types of values you might expect to use.